website/src/error-documentation.mjs

```diff
@@ -84,6 +81,16 @@ markdownParser.renderer.rules = {
   },
 };
 
+function marksHasBOM(codeHTML) {
+  const dom = new jsdom.JSDOM(codeHTML);
+  for (const mark of dom.window.document.querySelectorAll("mark")) {
+    if (mark.textContent?.includes("\u{feff}")) {
+      return true;
+    }
+  }
```

Comment:
Blocking: This checks if *any* `<mark>` starts with U+FEFF. We only want to check the *first* `<mark>` iff there is no text before the first `<mark>`.

Perhaps we should inspect `container.firstChild`. If it's a text node, check if it starts with U+FEFF. If it's a `<mark>`, check if its first child text node starts with U+FEFF.

Reply:
What you mean by `container.firstChild`?

Reply:
Block fixed